plugins/win-wasapi/win-wasapi.cpp

```diff
@@ -678,17 +716,16 @@ void WASAPISource::SetDefaultDevice(EDataFlow flow, ERole role, LPCWSTR id)
 	blog(LOG_INFO, "[WASAPISource::SetDefaultDevice][%08X] Default %s device changed",
 	     this, isInputDevice ? "input" : "output");
 
-	/* reset device only once every 300ms */
-	uint64_t t = os_gettime_ns();
-	if (t - lastNotifyTime < 300000000)
```

Comment:
Is it intended to remove this? If so, should we also remove `uint64_t lastNotifyTime = 0;`?